gradient/api_sdk/clients/deployment_client.py

```diff
@@ -33,12 +51,45 @@ def create(self, deployment_type, model_id, name, machine_type, image_url, insta
         return id_
 
     def start(self, deployment_id):
+        """
+        Start deployment
+
+        *EXAMPLE*::
+
+            gradient deployments start --id <your-deployment-id>
+
+        :param str deployment_id: Deployment ID
+        """
         return self._get_post_response(deployment_id)
 
     def stop(self, deployment_id):
+        """
+        Stop deployment
+
+        *EXAMPLE*::
+
+            gradient deployments stop --id <your-deployment-id>
+
+        :param deployment_id: Deployment ID
+        """
         return self._get_post_response(deployment_id, is_running=False)
 
     def list(self, filters):
+        """
+        List deployments with optional filtering
+
+        To view all running deployments in your team, run::
+
+            gradient list --state RUNNING
+
+        Options::
+
+          --state [BUILDING|PROVISIONING|STARTING|RUNNING|STOPPING|STOPPED|ERROR] Filter by deployment state
+          --projectId TEXT Use to filter by project ID
+          --model_id TEXT Use to filter by model ID
```

Comment:
is this correct? or should it be `modelId`?

Reply:
yes, I changed it.